change/@fluentui-react-charting-b5351308-a8f4-40d1-aebf-505c5d0ade20.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
mark type and dependentChangeType as null

Reply:
Ok